Base/usr/share/man/man7/Tips-and-Tricks.md

```diff
@@ -0,0 +1,63 @@
+![Welcome Icon](/res/icons/32x32/app-welcome.png)
+
+# Tips and Tricks
+
+This is a list of useful tips and tricks to help you make the most out of SerenityOS
+
+Related: [Keyboard Shortcuts](help://man/7/KeyboardShortcuts)
+
+## General
+* When on the Desktop or in File Manager, start typing and the nearest match will be selected
+* Bold text in context menus hints at the default behavior of a double-click
+* Hold `Ctrl` to accelerate mouse wheel interaction with sliders and spin boxes
+* Hold `Ctrl` while activating a menu item to prevent that menu from closing
+* Press `Ctrl+Shift+A` on a focused widget or application to show the Command Palette, a searchable list of available actions
+* Many applications will open a compatible file if you drag-and-drop it into their window
+* Default file and protocol associations can be changed in `~/.config/LaunchServer.ini`
+
+### Window Management
+* Double-click a window's title bar to maximize it
+* Click on a window's icon to open the context menu
+* Double-click on the edge of an application's window to maximize it in that direction
+* Middle-click on a window's maximize button to extend the window vertically (this can be undone the same way)
+* Resizable windows can be snapped to all edges of the screen. Drag a window to an edge or press `Super+Left, Right or Up` while it has focus
+* Cycle focus between windows by pressing and holding `Super+Tab`
+    * Use `Shift+Super+Tab` to do this in the reverse order
+* Move windows by grabbing anywhere within the window using `Super+Left-click`
+* Resize windows from anywhere within the window using `Super+Right-click`. They will resize in the direction you drag
+* Switch workspaces by pressing `Ctrl+Alt+Arrows`
+    * Use `Shift+Ctrl+Alt+Arrows` to switch workspace and bring the active window along too
```

Comment:
All already part of the shortcuts page

Reply:
I appreciate there's overlap between 'Keyboard Shortcuts' and 'Tips and Tricks'.
Originally the idea was that Keyboard Shortcuts was a cheatsheet, a quick reference, whereas Tips and Tricks could give a more detailed explanation of, for example, all of the methods for window management.
Nevertheless I'm removing the keyboard shortcut specific tips and will keep them just in Keyboard Shortcuts